lib/helpers/DataTypeUiHelper.ts

```diff
@@ -636,4 +682,123 @@ export class DataTypeUiHelper extends UiBaseLocators {
+  }
+
+  async doesAmountContainErrorMessageWitText(errorMessage: string) {
+    await expect(this.amountErrorMessage).toContainText(errorMessage);
+  }
+
+  async clickSingleBlockMode() {
+    // We need to wait, otherwise this would be flaky
+    await this.singleBlockModeBtn.waitFor();
+    await this.singleBlockModeBtn.click({force: true});
+  }
+
+  async clickLiveEditingMode() {
+    await this.liveEditingModeBtn.click({force: true});
+  }
+
+  async clickInlineEditingMode() {
+    await this.inlineEditingModeBtn.click({force: true});
+  }
+
+  async enterPropertyEditorWidth(width: string) {
+    await this.propertyEditorWidthInput.clear();
+    await this.propertyEditorWidthInput.fill(width)
+  }
+
+  async goToBlockWithName(name: string) {
+    await this.page.getByRole('link', {name: name}).click();
+  }
+
+  async enterBlockLabelText(label: string) {
+    await this.labelTextInput.clear();
+    await this.labelTextInput.fill(label);
+  }
+
+  async updateBlockOverlaySize(size: string) {
+    await this.overlaySizeOption.selectOption(size);
+  }
+
+  async addBlockContentModel(elementName: string) {
+    await this.chooseContentModelBtn.click();
+    await this.clickButtonWithName(elementName);
+    await this.clickChooseButton();
+  }
+
+  async addBlockSettingsModel(elementName: string) {
+    await this.chooseSettingsModelBtn.click();
+    await this.clickButtonWithName(elementName);
+    await this.clickChooseModalButton();
+  }
+
+  async removeBlockContentModel() {
+    await this.contentModelNode.hover();
+    await this.removeExactContentModelNodeBtn.click();
+  }
+
+  async removeBlockSettingsModel() {
+    await this.settingsModelNode.hover();
+    await this.removeExactSettingsModelNodeBtn.click();
+  }
+
+  async openBlockContentModel() {
+    await this.contentModelNode.hover();
+    await this.openBtn.click();
+  }
+
+  async openBlockSettingsModel() {
+    await this.settingsModelNode.hover();
+    await this.openBtn.click();
+  }
+
+  async isElementWorkspaceOpenInBlock(elementTypeName: string) {
+    await expect(this.page.locator('[alias="Umb.Workspace.DocumentType"]').filter({hasText: elementTypeName})).toBeVisible();
+  }
+
+  async enterBlockBackgroundColor(color: string) {
+    await this.backgroundColorInput.clear();
+    await this.backgroundColorInput.fill(color);
+  }
+
+  async enterBlockIconColor(color: string) {
+    await this.iconColorInput.clear();
+    await this.iconColorInput.fill(color);
+  }
+
+  async chooseBlockCustomStylesheetWithName(name: string) {
+    await this.chooseCustomStylesheetBtn.click();
+    await this.clickCaretButtonForName('wwwroot');
+    await this.clickCaretButtonForName('css');
+    await this.page.getByLabel(name, {exact: true}).click();
+    await this.clickChooseModalButton();
+  }
+
+  async clickRemoveCustomStylesheetWithName(name: string) {
+    await this.page.locator('[label="Custom stylesheet"]').locator('[name="' + name + '"]').click();
```

Comment:
this.page.locator('[label="Custom stylesheet"]') should be declared in constructor